styleSheet.css

```diff
@@ -258,36 +252,18 @@ ul {
   margin-top: 0;
 }
 
-.professional__list {
-  position: absolute;
-  top: 30%;
-}
-
-.professional__list > ul {
-  margin-left: 4%;
-  text-align: left;
-  top: 20%;
-}
-
-.professional__list li {
-  display: inline-flex;
-  margin: 10px 0;
-}
-
-.professional__percentBar {
-  background-color: var(--light_secondary);
-  height: 15px;
-  margin: 0% 5px 0 10px;
-  transform: translateY(50%);
-  width: 70vw;
-}
-
 #experince {
   background-image: url('images/experinceSection_backgroudImage.jpg');
   background-position: center top;
   background-repeat: no-repeat;
   background-size: cover;
-  height: 200vh;
+  height: calc(60em + 40vh);
+}
+
+.experince__backgroundOpacity {
```

Comment:
I think opacity could be a modifier or maybe you could leave it with just `.experince__background`, this way if you decide to change the style in the future you will not have to change the name

Reply:
I think I will leave it as `.experince__background`.